examples/SSD1306Demo/SSD1306Demo.ino

```diff
@@ -26,10 +26,14 @@ See more at http://blog.squix.ch
 #include "SSD1306.h"
 #include "SSD1306Ui.h"
 #include "images.h"
+#include "prototypes.h"
+
+// Defaulting the OLED connection to GPIO0 and GPIO2 pins
+#define SDA_PIN 0
+#define SDC_PIN 2
```

Comment:
What I noticed is that the pins are called SDA and SDC. But the correct naming for I2C is SDA and SCL.

Also, pins_arduino.h already defines the SDA and SCL values. I see it for the all variants.

```
static const uint8_t SDA = 4;
static const uint8_t SCL = 5;
```

So, you could write:

```
#define SDA_PIN    SDA
#define SCL_PIN    SCL
```
But really, you could get rid of the defines completely.